app.js

```diff
@@ -1,12 +1,15 @@
 const express = require("express");
 const { getCategories } = require("./controllers/categories-controllers.js");
+const { getReviews } = require("./controllers/reviews-controllers.js");
 const { handle500Statuses } = require("./controllers/error-handling-controllers.js");
 
 const app = express();
 
```

Comment:
I would add some error handling here for an invalid path - 404 :) I would also make sure you add a test for this under a new describe block (maybe for errors?)

Reply:
You can handle a general 404 in your app.js by setting the path to /* - have a look into this and try to add it :)